tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -760,6 +763,8 @@ private void createWildFlyScript() throws Exception {
             if (summaryMode) {
                 summaryString.append(String.format("Configured script for WildFly named %s.cli at %s.", fileSystemRealmName, fullOutputPath));
                 summaryString.append(System.getProperty("line.separator"));
+                summaryString.append(String.format("Script path is: %s.", scriptPath.toString()));
+                summaryString.append(System.getProperty("line.separator"));
```

Comment:
```suggestion
                summaryString.append(String.format("Configured script for WildFly at %s.", scriptPath.toString()));
                summaryString.append(System.getProperty("line.separator"));
```

The original script message can be modified to print the new path; we don't need to print both versions.